types/reply.d.ts

```diff
@@ -38,7 +38,7 @@ export interface FastifyReply<
   send(payload?: ReplyType): FastifyReply<RawServer, RawRequest, RawReply, RouteGeneric, ContextConfig, SchemaCompiler, TypeProvider>;
   header(key: string, value: any): FastifyReply<RawServer, RawRequest, RawReply, RouteGeneric, ContextConfig, SchemaCompiler, TypeProvider>;
   headers(values: {[key: string]: any}): FastifyReply<RawServer, RawRequest, RawReply, RouteGeneric, ContextConfig, SchemaCompiler, TypeProvider>;
-  getHeader(key: string): string | undefined;
```

Comment:
<img width="335" alt="image" src="https://user-images.githubusercontent.com/5475069/180979653-15e86f4f-352f-46ae-880c-6c1d367604c6.png">
Why not use `any` here? In nodejs doc, it returns is `any` type